examples/workload_specifications.yml

```diff
@@ -1,7 +1,4 @@
-min_input_tokens: 500
-max_input_tokens: 500
-min_output_tokens: 50
-max_output_tokens: 50
-frac_greedy: 1.0
-sample_size: 1
+input_tokens: 500
+output_tokens: 50
+greedy: True
```

Comment:
These changes belong in a different PR I think

Reply:
i actually created 2 branches. let me see how i can fix it.

Reply:
ok - let me know if i can help